tests/PHPStan/Analyser/data/array-filter-callables.php

```diff
@@ -0,0 +1,22 @@
+<?php // lint >= 7.4
```

Comment:
Is this file 7.4+ only? If so, you need to adjust NodeScopeSolverTest, as its not within the 7.4 only branch

Reply:
Oh I meant the other one with arrow functions.

Reply:
Oh god.. thx you too